### PR TITLE
Fix invalid glusterfs-daemonset.yaml

### PR DESCRIPTION
**Pull request by grebois**
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/345)

### CODE REVIEW

**Comment by SaravanaStorageNetwork** [comment]
@jarrpa  Please check and merge - I introduced this :(

File: deploy/kube-templates/glusterfs-daemonset.yaml [hunk 1]
```diff
@@ -25,8 +25,8 @@ spec:
         name: glusterfs
         env:
         - name: GB_GLFS_LRU_COUNT
-          value: 15
-        - name: TCMU_LOGDIR
+          value: "15"
+        - name: TCMU_LOG_DIR
```
Comment (jarrpa) [inline deploy/kube-templates/glusterfs-daemonset.yaml]: This var name has changed, please make sure it stays as `TCMU_LOGDIR`

**Comment by jarrpa** [comment]
Also, please squash all changes into a single commit.

File: deploy/kube-templates/glusterfs-daemonset.yaml (hunk 1 above)
Comment (grebois) [reply deploy/kube-templates/glusterfs-daemonset.yaml]: Sorry, missed that.

**Comment by grebois** [comment]
@jarrpa fixed and squashed, let me know.

**Comment by SaravanaStorageNetwork** [comment]
@grebois  Have you closed it without merge?   @jarrpa

**Comment by grebois** [comment]
@SaravanaStorageNetwork did not close it, at least not on purpose.

**Comment by SaravanaStorageNetwork** [comment]
@grebois  :)  - No issues. I have posted a PR for same and it is merged now

**Comment by jarrpa** [comment]
@grebois We went ahead and merged #346 while we were waiting, sorry.

**Pull request closed without merging**